src/args.rs

```diff
@@ -113,11 +142,11 @@ pub fn parse_args(mut args: Vec<OsString>) -> Result<ExecutionPlan, MagickError>
             // A file named "-foobar.jpg" will be parsed as an option.
             // Sadly imagemagick does not support the -- convention to separate options and filenames,
             // and there is nothing we can do about it without introducing incompatibility in argument parsing.
-            let (_sign, string_arg) = sign_and_arg_name(raw_arg)?;
+            let (sign, string_arg) = sign_and_arg_name(raw_arg)?;
             let arg = Arg::try_from(string_arg.as_str())
```

Comment:
This is an existing issue, but this should probably check the sign as well to avoid allowing e.g. `+auto-orient`.